### PR TITLE
misc: add remote visual studio code support to gceworker

### PR DESCRIPTION
**Pull request by cucaroach**
gceworker.sh code will start the gceworker, touch /.active, start code
and run a remote SSH to it.   When code exits the .active will be
removed.

Release note: none

### CODE REVIEW

File: scripts/gceworker.sh [hunk 1]
```diff
@@ -146,6 +150,13 @@ case "${cmd}" in
       -ignore 'Name *.o' \
       -ignore 'Name zcgo_flags*.go'
     ;;
+    code)
```
Comment (erikgrinaker) [inline scripts/gceworker.sh:153]: Consider `vscode` to make it clearer what it's for.

**Review by erikgrinaker — APPROVED** [review]
Nifty, thanks.

I suppose this means that the gceworker will keep running if the laptop is suspended while VSCode is open. No complaints from me, that doesn't seem terrible for the few people who use gceworkers as their main dev environment. But I've been discussing with @dt in the past some sort of dead man's switch that either keeps an open SSH session or pings the gceworker regularly, causing it to shut down when the laptop suspends, and then automatically start it back up when the laptop resumes. Just a thought.

**Review by cucaroach — COMMENTED** [review]
Cool, yeah something like that would probably be overkill for me, this just addresses I shutdown vscode but left the gceworker running problem which I do ALL THE TIME!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/73336#-:-MpqpJEODlN-vrCdAguc:bvq0rqv)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @erikgrinaker)

___
*[scripts/gceworker.sh, line 153 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/73336#-MpqPg-rLXBh2yMNp9T1-r1-153:-MpqpFbH1kaTRDpdh11q:b-8fx2u7) ([raw file](https://github.com/cockroachdb/cockroach/blob/9b9f56b09a3324430c27bab30dc67e4ebece2cd1/scripts/gceworker.sh#L153)):*
<details><summary><i>Previously, erikgrinaker (Erik Grinaker) wrote…</i></summary><blockquote>

Consider `vscode` to make it clearer what it's for.
</blockquote></details>

:+1:

**Comment by dt** [comment]
Yeah, as mentioned, I use a something slightly different. I tend to leave vscode running for days or weeks at a time, so something like this, that only stops my worker when vscode exits, would basically be the same as just touching .active once and then never stopping it. 

Instead, I leave something like `while true; do ./scripts/gceworker.sh start -- read i; sleep 10; done` running in the background. This basically means any time my laptop wakes up it will start my gceworker, and the `read` will just block, leaving that ssh session open. The open ssh session prevents worker shutdown, but unlike `/.active`, it is tied to my laptop actually being connected -- if I close my laptop for the night (without closing vscode), my worker stops as well.

**Comment by dt** [comment]
Huh, actually, now I'm wondering if this is just based on keeping the workup up until vscode quits, what does touching .active do that vscode's own ssh session to the remote server process doesn't? or do you want your worker to stay up even if your client is off/disconnected but vscode hasn't been terminated?

**Review by cucaroach — COMMENTED** [review]
I didn't realize the vscode ssh session was enough to keep the worker up, I guess before I was manually touch'ing .active before so I just wanted to automate deleteing it, but for my use cases I guess I can just remove the .active bits and forget about it.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/73336#-:-Mpr9_fg9Ql9H2Hngh0q:bgq623m)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @erikgrinaker)

**Pull request merged**